src/renderer/views/MainRouteRenderer.scss

```diff
@@ -80,7 +80,16 @@
 
     .header {
         grid-area: top-bar;
-        margin: 0;
+        margin-inline: -10px;
+        margin-block-start: -10px;
+        background-color: #202020;
+
+        svg {
```

Comment:
this is too wide of a selector, it affects ALL svgs in the header. may want to increase specificity